docs/user-guides/integration-istio.md

```diff
@@ -0,0 +1,119 @@
+
+### 1.3 Install Istio DNS
+As any Servicecomb service center service will be translated to Serviceentry in K8S, while Kubernetes provides DNS resolution for Kubernetes Services out of the box, any custom ServiceEntrys will not be recognized. In addition to capturing application traffic, Istio can also capture DNS requests to improve the performance and usability of your mesh
+
+Use the following command to install istio DNS:
+```
+cat <<EOF | istioctl install -y -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      proxyMetadata:
+        # Enable basic DNS proxying
+        ISTIO_META_DNS_CAPTURE: "true"
+        # Enable automatic address allocation, optional
+        ISTIO_META_DNS_AUTO_ALLOCATE: "true"
+EOF
+```
+
+### 1.4 Install Servicecomb service center
+Servicecomb service center could be installed in K8S or on VM. 
+Install Servicecomb service center follow this [instruction](https://github.com/apache/servicecomb-service-center/blob/master/README.md)
+
+## 2 Install Servicecomb-service-center-istio
+### 2.1 Building
+You don’t need to build from source to use Servicecomb-service-center-istio (binaries in apache nexus ), but if you want to try out the latest and greatest, Servicecomb-service-center-istio can be easily built.
+```
+go build -o servicecomb-service-center-istio cmd/main.go
+```
+
+### 2.2 Building docker image
+```
+docker build -t servicecomb-service-center-istio:dev .
+```
+
+### 2.2 Run on VM
+```
+./servicecomb-service-center-istio --sc-addr=?SERVICE_CENTER_ADDRESS --kube-config=?KUBE_CONFIG_FILE_PATH
+```
+
+### 2.3 Run in K8S
+```
+# make sure you modified the input args in the deployment.yaml file first, specify you service center address
+kubectl apply -f manifest/deployment.yaml
+```
+
+### 2.4 Input parameters
+![image](integration-istio.png)
+
+## 3 Best Practices
+We will use [consumer-provider](../../istio/examples/consumer-provider/) example to show how to use this tool.
+
+We have two services: Provider and Consumer:
+* `provider` is the provider of a service which calculates and returns the sum of the square root from 1 to a user provided parameter `x`.
+* `consumer` performs as both provider and consumer. As a consumer, it calls the api provided by `provider` to get the result of the sum of square roots;
+as a `provider`, it provides a service externally which returns the result it gets from `provider` to its clients.
+
+While Provider uses servicecomb service center tech stack, Consumer uses istio tech stack. Origionaly, Provider and Consumer couldn't discover each other. 
+
+In this demo, we are going to adopt our servicecomb-service-center-istio to brake the barrier between Provider and Consumer.
+
+The logical network and calling relationship of the Provider and Consumer is as follows：
+
+![image](integration-istio-demo.png)
+
+### 3.1 Build Provider and Consumer service images
```

Comment:
3.1 chapter should name as "Example"， and simply draw a toplogy of this example along with istio and service center

Reply:
I removed the image here as I have already clearly described each component in this example